### DISCUSSION TITLE
Dutch translation

### DISCUSSION THREAD

**Discussion by meniv-nl** [malzariey/filament-daterangepicker-filter#27] [General]
Hi There, 

Great work! Hopefully fast a upgrade to filament v3.
Because I don't have persmissions to push/pull/commit etc. I'll leave the dutch (nl) translation here:
[message.php.gz](https://github.com/malzariey/filament-daterangepicker-filter/files/12253342/message.php.gz)

**Comment by malzariey** [comment]
You could have made a fork then pull request to submit the translation. 

Thank you for your contribution. 

I have added the nl language file.
